visual.go

```diff
@@ -0,0 +1,156 @@
+package main
+
```

Comment:
Just wondering if it might be safer to match on just the path instead of the entire URL also means you are explicitly ignore any request parameters.

Reply:
not sure which bit this comment refers to?